packages/graphback-codegen-resolvers/src/api/targetResolverContext.ts

```diff
@@ -180,10 +181,11 @@ function createRelations(inputContext: InputModelTypeContext[]) {
           });
         }
         else if (f.annotations.OneToMany || f.isArray) {
-          let columnName = `${f.name}Id`;
+          let columnName = `${lowerCaseFirstChar(t.name)}Id`;
```

Comment:
I guess this is temporary - when using annotations we need to actually check if there is metadata that affects that file

Reply:
`getRelationFieldName` method does that